pkgs/tools/text/shab/shab.nix

```diff
@@ -0,0 +1,66 @@
+{ bash, stdenv, lib, runCommand, writeText, fetchFromGitHub }:
+let
+  version = "1.0.0";
+
+  shab = stdenv.mkDerivation {
```

Comment:
A `meta` would be nice.

Reply:
good call, fixed in 2824cd8b6f9b8e200d1187f6369fd3fe6e815ab0